src/torchrunx/launcher.py

```diff
@@ -83,6 +72,7 @@ class Launcher:
     ssh_config_file: str | os.PathLike | None = None
     backend: Literal["mpi", "gloo", "nccl", "ucc", None] = None
     log_dir: os.PathLike | str = "./logs"
+    log_spec: dict[str, list[logging.Handler]] | None = None
```

Comment:
Probably change this to one variable: `logging: dict[str, list[logging.Handler]] | os.PathLike | str | None`
where `os.PathLike | str` are a directory.

Reply:
We might also want to make `dict[str, list[logging.Handler]]` it's own class (like an abstraction)

Reply:
I've taken a crack at this, lmk what you think